chi-immigration-task-force/icirr-app
# Map Route (with clustering!)

**Test Plan**
- Make sure all 50 markers render with no javascript errors
- Make sure zooming out shows a cluster of 50
- Make sure clicking on a marker pans to and opens its info panel
- Make sure the clusters break up as expected
